definitions/json/CaseField.json

```diff
@@ -3106,6 +3106,118 @@
     "FieldType": "Document",
     "SecurityClassification": "Public"
   },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "ID": "hearingAndReferralDetails",
+    "Label": "Hearing and Referral Details placeholder",
+    "FieldType": "Text",
+    "SecurityClassification": "Public"
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "ID": "hearingAndReferralDetailsLabel",
+    "Label": "${hearingAndReferralDetails}",
+    "FieldType": "Label",
+    "SecurityClassification": "Public"
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "ID": "directionTo",
+    "Label": "Who are you sending these directions to?",
+    "FieldType": "FixedRadioList",
+    "FieldTypeParameter": "frl_ReferCaseTo",
+    "SecurityClassification": "Public"
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "ID": "replyToEmailAddress",
+    "Label": "What is their email address?",
+    "FieldType": "Text",
+    "SecurityClassification": "Public"
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "ID": "isUrgentReply",
+    "Label": "Is this urgent?",
+    "FieldType": "YesOrNo",
+    "SecurityClassification": "Public"
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "ID": "directionDetails",
+    "Label": "What are your directions?",
+    "HintText": "Give details of your reply to the referral and any issues you've identified",
+    "FieldType": "TextArea",
+    "SecurityClassification": "Public"
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "ID": "replyDocument",
+    "Label": "Upload document",
+    "FieldType": "Collection",
+    "FieldTypeParameter": "ReplyToReferralDocumentUpload",
```

Comment:
Any reason these are their own type and not using DocumentUpload type?

Reply:
This field doesn't have the document type section: 
![image](https://user-images.githubusercontent.com/101253854/186651583-b7ec4196-b5fc-4198-9ebf-d068b2227e48.png)

Reply:
But using EventToComplexTypes, you can hide 'Document Type' field